internal/blocktx/store/postgresql/postgres_test.go

```diff
@@ -309,36 +373,149 @@ func TestPostgresDB(t *testing.T) {
-			require.Equal(t, uint64(826481), tx.BlockHeight)
+	t.Run("get registered txs", func(t *testing.T) {
+		// given
+		prepareDb(t, postgresDB, "fixtures/get_transactions")
+
+		blockHash := testutils.RevChainhash(t, "000000000000000005aa39a25e7e8bf440c270ec9a1bd30e99ab026f39207ef9")
+		blockHash2 := testutils.RevChainhash(t, "0000000000000000072ded7ebd9ca6202a1894cc9dc5cd71ad6cf9c563b01ab7")
+
+		expectedTxs := []store.TransactionBlock{
+			{
+				TxHash:      testutils.RevChainhash(t, "21132d32cb5411c058bb4391f24f6a36ed9b810df851d0e36cac514fd03d6b4e")[:],
+				BlockHash:   blockHash[:],
+				BlockHeight: 822013,
+				MerklePath:  "merkle-path-2",
+				BlockStatus: blocktx_api.Status_LONGEST,
+			},
+			{
+				TxHash:      testutils.RevChainhash(t, "213a8c87c5460e82b5ae529212956b853c7ce6bf06e56b2e040eb063cf9a49f0")[:],
+				BlockHash:   blockHash2[:],
+				BlockHeight: 822012,
+				MerklePath:  "merkle-path-6",
+				BlockStatus: blocktx_api.Status_STALE,
+			},
+			{
+				TxHash:      testutils.RevChainhash(t, "12c04cfc5643f1cd25639ad42d6f8f0489557699d92071d7e0a5b940438c4357")[:],
+				BlockHash:   blockHash2[:],
+				BlockHeight: 822012,
+				MerklePath:  "merkle-path-7",
+				BlockStatus: blocktx_api.Status_STALE,
+			},
 		}
+
+		// when
+		actualTxs, err := postgresDB.GetRegisteredTxsByBlockHashes(ctx, [][]byte{blockHash[:], blockHash2[:]})
+
+		// then
+		require.NoError(t, err)
+		require.Equal(t, expectedTxs, actualTxs)
+	})
+
+	t.Run("get registered txs by block hashes", func(t *testing.T) {
+		// given
+		prepareDb(t, postgresDB, "fixtures/get_transactions")
+
+		blockHashLongest := testutils.RevChainhash(t, "000000000000000005aa39a25e7e8bf440c270ec9a1bd30e99ab026f39207ef9")
+		blockHashStale := testutils.RevChainhash(t, "0000000000000000072ded7ebd9ca6202a1894cc9dc5cd71ad6cf9c563b01ab7")
+
+		blockHashes := [][]byte{
+			blockHashLongest[:],
+			blockHashStale[:],
+		}
+
+		expectedTxs := []store.TransactionBlock{
+			{
+				TxHash:      testutils.RevChainhash(t, "21132d32cb5411c058bb4391f24f6a36ed9b810df851d0e36cac514fd03d6b4e")[:],
+				BlockHash:   blockHashLongest[:],
+				BlockHeight: 822013,
+				MerklePath:  "merkle-path-2",
+				BlockStatus: blocktx_api.Status_LONGEST,
+			},
+			{
+				TxHash:      testutils.RevChainhash(t, "213a8c87c5460e82b5ae529212956b853c7ce6bf06e56b2e040eb063cf9a49f0")[:],
+				BlockHash:   blockHashStale[:],
+				BlockHeight: 822012,
+				MerklePath:  "merkle-path-6",
+				BlockStatus: blocktx_api.Status_STALE,
+			},
+			{
+				TxHash:      testutils.RevChainhash(t, "12c04cfc5643f1cd25639ad42d6f8f0489557699d92071d7e0a5b940438c4357")[:],
+				BlockHash:   blockHashStale[:],
+				BlockHeight: 822012,
+				MerklePath:  "merkle-path-7",
+				BlockStatus: blocktx_api.Status_STALE,
+			},
+		}
+
+		// when
+		actualTxs, err := postgresDB.GetRegisteredTxsByBlockHashes(ctx, blockHashes)
+
+		// then
+		require.NoError(t, err)
+		require.Equal(t, expectedTxs, actualTxs)
 	})
```

Comment:
This seems to be the exact same test case as the previous one "get registered txs".
I think one of them could be removed

Reply:
You're right, the first one should be removed, thanks for spotting that!